packages/gui/src/electron/main.tsx

```diff
@@ -29,14 +30,21 @@ import About from '../components/about/About';
 import packageJson from '../../package.json';
 import AppIcon from '../assets/img/chia64x64.png';
 import windowStateKeeper from 'electron-window-state';
+import validateSha256 from './validateSha256';
 
 const NET = 'mainnet';
 
 app.disableHardwareAcceleration();
+app.commandLine.appendSwitch('disable-http-cache');
```

Comment:
why we are doing it?

Reply:
We can disable the HTTP cache when our own caching is in place.